### DISCUSSION TITLE
Digitally signed versoin of Svg.dll

### DISCUSSION THREAD

**Discussion by ZubairImran** [vvvv/SVG#1167] [Q&A]
Hi Team,
When i download the svg.dll its not digitally signed. Is there any svg.dll available with digital signature. Or is it ok to sign it and use it in my application

**Comment by ArBom** [comment]
Just try download it, and use it from NuGet. [All packages uploaded to nuget.org are automatically repository signed.](https://learn.microsoft.com/en-us/nuget/reference/signed-packages-reference)

**Comment by ZubairImran** [comment]
Thank you, will try the above step
